bin/ci

```diff
@@ -51,8 +51,12 @@ function update_sources() {
             template=".circleci/src/workflows/site_no_change.yml.tpl"
         fi
 
+        # Look for custom image name in site's setup.cfg file and default to the site name
+        image=$(grep -Po '(?<=^name = ).*' "${site_path}/src/backend/setup.cfg" || echo "${site}")
```

Comment:
Wouldn't it be easier to read with a grep/sed combo instead of a PCRE?

Reply:
Or even grep/cut:

```suggestion
        image=$(grep "^name = " "${site_path}/src/backend/setup.cfg" | cut -c 7- || echo "${site}")
```

Reply:
you're right it's more readable!